app/addons/databases/components.react.jsx

```diff
@@ -0,0 +1,271 @@
+
+    createRows: function () {
+      return _.map(this.props.body, function (item, iteration) {
+        return (
+          <DatabaseRow row={item} />
+        );
+      });
+    },
+
+    render: function () {
+      var rows = this.createRows();
+      return (
+        <div className="view">
+          <table className="databases table table-striped">
+            <thead>
+              <th>Name</th>
+              <th>Size</th>
+              <th># of Docs</th>
+              <th>Update Seq</th>
+              <th>Actions</th>
+            </thead>
+            <tbody>
+            {rows}
+            </tbody>
+          </table>
+        </div>
+      );
+    }
+  });
+
+  var DatabaseRow = React.createClass({
+
+    renderGraveyard : function (row) {
+      if (row.status.isGraveYard()) {
+        return (
+          <GraveyardInfo row={row} />
+        );
+      } else {
+        return null;
+      }
+    },
+
+    render: function () {
+      var row = this.props.row;
+      var name = row.get("name");
+      var encoded = app.utils.safeURLName(name);
+      var size = Helpers.formatSize(row.status.dataSize());
+      return (
+        <tr>
+          <td>
+            <a href={"#/database/"+encoded+"/_all_docs"}>{name}</a>
+          </td>
+          <td>{size}</td>
+          <td>{row.status.numDocs()} {this.renderGraveyard(row)}</td>
+          <td>{row.status.updateSeq()}</td>
+          <td>
+            <a className="db-actions btn fonticon-replicate set-replication-start" title={"Replicate "+name} href={"#/replication/"+encoded}></a>&#160;
+            <a className="db-actions btn icon-lock set-permissions" title={"Set permissions for "+name} href={"#/database/"+encoded+"/permissions"}></a>
+          </td>
+        </tr>
+      );
+    }
+  });
+
+  var GraveyardInfo = React.createClass({
+
+    componentDidMount : function () {
+      $(this.refs.myself.getDOMNode()).tooltip();
+    },
+
+    render : function () {
+      var row = this.props.row;
+      return (
+        <i className="js-db-graveyard icon icon-exclamation-sign" ref="myself" title={"This database has just " + row.status.numDocs() + " docs and " + row.status.numDeletedDocs() + " deleted docs"}></i>
+      );
+    }
+  });
+
+  var RightDatabasesHeader = React.createClass({
+
+    render : function () {
+      return (
+        <div className="header-right">
+          <AddDatabaseWidget />
+          <JumpToDatabaseWidget />
+        </div>
+      );
+    }
+  });
+
+  var AddDatabaseWidget = React.createClass({
+
+    onTrayToggle : function () {
+      var that = this;
+      this.refs.newDbTray.toggle(function (shown) {
+        if (shown) {
+          that.refs.newDbName.getDOMNode().focus();
+        }
+      });
+    },
+
+    onKeyUpInInput : function (e) {
+      if (e.which === 13) {
+        this.onAddDatabase();
+      }
+    },
+
+    onAddDatabase : function () {
+      var databaseName = $(this.refs.newDbName.getDOMNode()).val();
```

Comment:
that is how we would do it in jquery, take a look at https://github.com/apache/couchdb-fauxton/blob/master/app/addons/cors/components.react.jsx#L69 for how to solve that in react components without jquery